test/src/test-helpers.ts

```diff
@@ -0,0 +1,103 @@
+import * as childProcess from "child_process";
+import * as cps from "current-processes";
+
+export async function closeDesktopApplication(application: string): Promise<boolean> {
+    return new Promise<boolean>(async function (resolve, reject) {
```

Comment:
I don't think we need closeDesktopApplication, getProcessId, or executeCommandLine for Excel Custom Functions since we close the workbook via the Excel API